app/main.py

```diff
@@ -1,13 +1,25 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        key: value for key, value in {
+            "line": error["line_number"],
+            "column": error["column_number"],
+            "message": error["text"],
+            "name": error["code"],
+            "source": "flake8"
+        }.items()
+    }
```

Comment:
```suggestion
    return {
        "line": error["line_number"],
        "column": error["column_number"],
        "message": error["text"],
        "name": error["code"],
        "source": "flake8"
    }
```
no need to use loop here